extensions/override.c

```diff
@@ -230,12 +232,29 @@ hack_can_send(void *vdata)
 		if (MyClient(data->client))
 		{
 			update_session_deadline(data->client);
-			sendto_realops_snomask(SNO_GENERAL, L_NETWIDE, "%s is using oper-override on %s (forcing message)",
+			sendto_realops_snomask(SNO_GENERAL, L_NETWIDE, "%s is using oper-override on %s (invite: )",
```

Comment:
wut

Reply:
oh man I meant to undo that before I committed. egg on my face